src/material/checkbox/checkbox.ts

```diff
@@ -312,11 +312,11 @@ export abstract class _MatCheckboxBase<E>
   }
 
   _getAriaChecked(): 'true' | 'false' | 'mixed' {
```

Comment:
can this function just be removed or is it still used somewhere?

Reply:
good catch. `_getAriaChecked` is not used anymore, and the legacy checkbox doesn't use it, so I deleted it.